src/app/containers/Timestamp/index.jsx

```diff
@@ -2,14 +2,16 @@ import React from 'react';
 import { number } from 'prop-types';
 import Timestamp from '../../components/Timestamp';
 import relativeTime from './relativeTimestamp';
+import {
+  isValidDateTime,
+  leadingZero,
+  formatUnixTimestamp,
+  isTenHoursAgoOrLess,
+} from './timestampUtilities';
 import { GridItemConstrainedMedium } from '../../lib/styledGrid';
 
-// if the date is invalid return null - https://stackoverflow.com/questions/1353684/detecting-an-invalid-date-date-instance-in-javascript#answer-1353711
-const isValidDateTime = dateTime => !isNaN(dateTime); // eslint-disable-line no-restricted-globals
-
-const leadingZero = val => (val < 10 ? `0${val}` : val);
-
-const months = [ 'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December' ]; // eslint-disable-line
+// prettier-ignore
+const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']; // eslint-disable-line
 
 // 2019-03-22
 const longNumeric = {
```

Comment:
These are veerrrrry coupled with formatUnixTimestamp, can you import these in from another file?